### PR TITLE
Add getting started button

### PR DESCRIPTION
**Pull request by gtsiolis**
This will add a secondary button to link to the new [Getting Started](https://www.gitpod.io/docs/getting-started/) section in the docs which mentions how you can use Gitpod.

**Context**: This builds on top of the original proposal in https://github.com/gitpod-io/website/issues/882#issue-759375835 and moves us one step closer to https://github.com/gitpod-io/website/issues/876.

| BEFORE | AFTER |
|-|-|
| image | image |

### CODE REVIEW

**Comment by gtsiolis** [comment]
@JohannesLandgraf WDYT? Feel free to close this if you think this is going into the wrong direction. /cc @ChristinFrohne

File: src/components/index/Banner.tsx [hunk 1]
```diff
@@ -78,6 +78,9 @@ const Banner = () => (
             <a href="#get-started" className="btn btn--cta">
               Try Now
             </a>
+            <a href="https://www.gitpod.io/docs/getting-started/" className="btn" style={{ marginLeft: '8px' }}>
```
Comment (gtsiolis) [inline src/components/index/Banner.tsx:81]: **thought**: Alternatively, we could link to the [Introduction](https://www.gitpod.io/docs/) section, it just seemed natural and more practical to link to the [Getting Started](https://www.gitpod.io/docs/getting-started/) section. /cc @JohannesLandgraf

**Comment by ChristinFrohne** [comment]
I as a user would be confused about the differences between `try now` and `get started`. Instead, we could add the link to the documentation to https://www.gitpod.io/#get-started

File: src/components/index/Banner.tsx (hunk 1 above)
Comment (JohannesLandgraf) [reply src/components/index/Banner.tsx:81]: Agree, also the Getting Started Section links back to the Introduction Section so should be fine.

**Review by JohannesLandgraf — APPROVED** [review]
I really like the idea @gtsiolis, let's merge after you resolved the conflicts

**Comment by gtsiolis** [comment]
Merging per https://github.com/gitpod-io/website/pull/911#pullrequestreview-582184591. Thanks @JohannesLandgraf @ChristinFrohne for the feedback! 🙏 

@ChristinFrohne regarding https://github.com/gitpod-io/website/pull/911#issuecomment-747457299, this will probably become better and more clear after the redesign but I'd suggest to avoid removing the only primary action that prompts the user to immediately open a new environment using an example project from this section. Let's discuss this in a new issue if you think it's worth putting more effort in this before the redesign. 🌱

**Pull request merged**